_includes/footer.html

```diff
@@ -1,14 +1,79 @@
-{% capture phoneNumber %}tel{{ site.theme_settings.phone_number | replace: " ", "" }}{% endcapture %}
-{% if site.theme_settings.copyright %}
 <footer class="slice footer">
-    <p class="footer__container container">
-		<span class="footer__text">{{ site.theme_settings.copyright }} © {{ 'now' | date: "%Y" }}</span>
-		<span class="footer__text">
-			<a class="footer__link no-link-style" href="{{ phoneNumber }}">
-				{{ site.theme_settings.phone_number }}
+    <div class="footer__container">
+			<a class="footer__logo-eleven" href="{{ site.baseurl }}/">
+					{% include eleven_logo.html %}
 			</a>
-		</span>
-		<span class="footer__text">{{ site.theme_settings.adress }} {{ site.theme_settings.post_code }}</span>
-    </p>
+			<span class="footer__logo-title" >
+				{{ site.theme_settings.brand }}
+			</span>
+			<span class="footer__content">
+				{% translate global.discover_eleven %}
+			</span>
+			<span class="footer__subtitle">
+				{% translate global.know_us %}
+			</span>
+			<a class="footer__button" target="_blank" href="{{ site.theme_settings.site_web_eleven }}/">
+				{% translate global.go_website %}
+			</a>
+		</div>
+		<div class="footer__container-contact">
+			<span class="footer__title">
+				{% translate global.contact %}
+			</span>
+			<span class="footer__subtitle">
+				{% translate global.eleven_paris %}
+			</span>
+			<span class="footer__content">
+				{% translate global.eleven_address_paris %}
+			</span>
+			<div class="footer__content-links">
+				<a class="footer__social-link-rss" target="_blank" href="{{ site.baseurl }}/{{ site.theme_settings.rss }}">
+					{% include rss_logo.html %}
+				</a>
+				<a class="footer__social-link" target="_blank" href="{{ site.baseurl }}/{{ site.theme_settings.link_facebook }}">
+					{% include facebook_logo.html %}
+				</a>
+				<a class="footer__social-link" target="_blank" href="{{ site.baseurl }}/{{ site.theme_settings.link_twitter }}">
+					{% include twitter_logo.html %}
+				</a>
+				<a class="footer__social-link" target="_blank" href="{{ site.baseurl }}/{{ site.theme_settings.link_linkedin }}">
+					{% include linkedin_logo.html %}
+				</a>
+				<a class="footer__social-link" target="_blank" href="{{ site.baseurl }}/{{ site.theme_settings.link_wttj }}">
```

Comment:
```suggestion
				<a class="footer__social-link" target="_blank" href="{{ site.theme_settings.link_wttj }}">
```

sinon ca devient `http://dev.blog.eleven-labs.com.s3-website.eu-west-2.amazonaws.com/feat-new-design/https://www.welcometothejungle.com/fr/companies/eleven-labs`

Reply:
Modifié